src/mpyl/steps/deploy/k8s/resources/dagster.py

```diff
@@ -45,14 +63,17 @@ def to_user_code_values(
                     "repository": f"{docker_registry.host_name}/{project.name}",
                 },
                 "includeConfigInLaunchedRuns": {"enabled": True},
-                "name": f"{project.name}{name_suffix}",
+                "name": release_name,
                 "port": 3030,
                 "resources": {
                     "requests": {"memory": "256Mi", "cpu": "50m"},
                     "limits": {"memory": "512Mi", "cpu": "1000m"},
                 },
             },
         ],
+        "extraManifests": [to_dict(manifest) for manifest in extra_manifests]
```

Comment:
What would be the best practice here @koeves , to add an empty list as value or no key/value at all if there are no extra manifests?

Reply:
I guess it would look nicer if we don't write `extraManifests` there if the list is empty.. But then, no one is actually looking at this result yaml right? It's just for helm.

Reply:
yah helm doesnt care whether `extraManifests` exist or not.